### ISSUE TITLE
Update RDS docs page

### ISSUE THREAD

**Issue by mheadd** [cloud-gov/cg-site#1870]
The [RDS Services docs page](https://cloud.gov/docs/services/relational-database/) contains an outdated command for listing marketplace services (it might work with cf v6 but it definitely does not work with v7).

Currently:

```bash
cf marketplace -s aws-rds
```

Should be:

```bash
cf marketplace -e aws-rds
```

Additionally, the guidance on how to check which version of an RDS is deployed needs to be changed since customers [will likely not be able to query the AWS API for version information](https://gsa-tts.slack.com/archives/C09CR1Q9Z/p1607966074265000?thread_ts=1607706661.261700&cid=C09CR1Q9Z) 🔒. Unfortunately, the only way to determine the version of Postgres that is deployed seems to be to actually create an instance and then run `SELECT version();` after connecting to it.

**Comment by mheadd** [comment]
Related change [here](https://github.com/cloud-gov/cg-site/issues/1765).

**Comment by ccostino** [comment]
D'oh, you're right!  There is a difference between `cf` CLI v6 vs. v7:

- v6 only uses the [`-s` parameter](http://cli.cloudfoundry.org/en-US/v6/marketplace.html)
- v7 only uses the [`-e` parameter](http://cli.cloudfoundry.org/en-US/v7/marketplace.html)

Furthermore, it'd be nice if we were able to include the version number in the info returned when getting details about a service instance, either with the `cf` CLI (`cf service <SERVICE INSTANCE NAME>`) or using `cf curl` ([`cf curl /v3/services_instances/:guid`](http://v3-apidocs.cloudfoundry.org/version/3.93.0/#get-a-service-instance).

It looks like at least with the `cf curl` command for getting instance details in the V3 API there is a `maintenance_info` sub-object that includes a version number in the example there.  If we could at least expose that, it would be very helpful.  I took a quick look in the broker code and didn't see an obvious place where this would fit in, but I imagine there has to be a way.
